openedx/core/lib/docs/decisions/0002-logging-and-monitoring-expected-errors-removed.rst

```diff
@@ -0,0 +1,44 @@
+Logging and monitoring of expected errors removed
+=================================================
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+The setting ``EXPECTED_ERRORS`` was used for both expected and ignored errors in edx-platform.
+
+The functionality supporting expected errors was added:
+
+* Because New Relic didn't yet support this for Python, and
+* Because it was thought that people may want more flexibility for marking errors as expected in addition to just the class name, and
+* The setting provides a place to document why the change is being made.
+
+Updates regarding these three original reasons:
+
+* New Relic now supports expected errors, so the custom functionality is redundant and complicated, and
+* This functionality has yet to be used, so it is also unnecessary, and
```

Comment:
Missing a third list item.

Reply:
There was a third item in the DEPR, but I moved it to the consequences section, so I just dropped the ", and".